examples/manual_builder.rs

```diff
@@ -0,0 +1,33 @@
+use veil::redactor::{Redactor, RedactorBuilder};
+
+fn main() {
+    // Build a new Redactor.
+    // We'll set up the Redactor to use flags that are equivalent to:
+    // `#[redact(with = 'X', partial))]`
+    // on a field, when using the `Redact` derive macro.
+    let redactor: Redactor = RedactorBuilder::new().char('X').partial().build().unwrap();
+
+    // We can now redact any string we want in a number of different ways...
+
+    // Firstly, we can simply redact directly to a `String`:
+    assert_eq!(redactor.redact("Hello, world!".to_string()), "HelXX, XXrld!");
+
+    // Or, we can redact a `String` in-place, which is slightly more efficient,
+    // and we can also chain multiple redactions together:
```

Comment:
```suggestion
    // Or, we can redact a `String` in-place, which is slightly more efficient,
    // and allows us to chain multiple redactions together:
```
Something like this reads a bit better for me